onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -1383,10 +1401,11 @@ def aten_det(self: TensorType) -> TensorType:
     raise NotImplementedError()
 
 
+@torch_op("aten::detach")
 def aten_detach(self: TensorType) -> TensorType:
     # detach(Tensor(a) self) -> Tensor(a)
 
-    raise NotImplementedError()
+    return op.Identity(self)
```

Comment:
```suggestion
    return self
```

This should be good

Reply:
if return self only, no op.xxx() function called inside this function, the test cannot be passed.

Reply:
I see. SG